lib/utils/result_formatter.ts

```diff
@@ -284,14 +284,24 @@ export class ResultFormatter {
         });
     }
 
-    static off(decodeResult: DecodeResult, time: number) {
-        decodeResult.raw.off_time = time;
-        decodeResult.formatted.items.push({
-            type: 'time_of_day',
-            code: 'OFF',
-            label: 'Takeoff Time',
-            value: DateTimeUtils.timestampToString(time, 'tod'),
-        });
+    static off(decodeResult: DecodeResult, time: number, type: 'tod' | 'epoch' = 'tod') {
```

Comment:
i already do the `_date` and `_time` in eta - is this something we want to keep doing, or is there a better way?

Reply:
Hmmm, good question.

Reply:
After some thought, I think notating the type is helpful.